query/src/main/java/org/infinispan/search/mapper/impl/InfinispanMappingInitiator.java

```diff
@@ -34,7 +34,7 @@ public InfinispanMappingInitiator(PojoBootstrapIntrospector introspector,
                                      Collection<ProgrammaticSearchMappingProvider> mappingProviders,
                                      BlockingManager blockingManager, FailureCounter failureCounter,
                                      IndexerConfig indexerConfig) {
-      super(introspector);
+      super(introspector, () -> "Cannot read Jandex Root Mapping");
```

Comment:
It probably would be more future-proof to have the JBoss logging message bundle like we have in Search (https://github.com/hibernate/hibernate-search/blob/main/mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/reporting/impl/StandalonePojoMapperHints.java) as we might be adding more message methods, in particular, because of the changes for https://hibernate.atlassian.net/browse/HSEARCH-798